src/index.html

```diff
@@ -22,6 +22,378 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a
+          href="#"
+          class="header__logo-link"
+        >
+          <img
+            src="images/logo.png"
+            alt="MOYO logo"
+            class="header__logo-link-img"
+          />
+        </a>
+      </div>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__list-item">
+            <a
+              href="#apple"
+              class="header__list-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__list-item">
+            <a
+              href="#samsung"
+              class="header__list-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__list-item">
+            <a
+              href="#smartphones"
+              class="header__list-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__list-item">
+            <a
+              href="#laptops_computers"
+              class="header__list-link"
+              data-qa="card-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__list-item">
+            <a
+              href="#gadgets"
+              class="header__list-link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__list-item">
+            <a
+              href="#tablets"
+              class="header__list-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__list-item">
+            <a
+              href="#photo"
+              class="header__list-link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__list-item">
+            <a
+              href="#video"
+              class="header__list-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="catalog-wrapper">
+      <main class="main">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="images/imac.jpeg"
+            alt="iMac photo"
+            class="card__img"
```

Comment:
alt attribute should describe the image content
![image](https://github.com/mate-academy/layout_catalog/assets/106397016/013a666a-ddf1-4528-bd97-59668fcbab39)